src/tiles.js

```diff
@@ -0,0 +1,146 @@
+import keys from 'lodash/keys';
+import includes from 'lodash/includes';
+import isUndefined from 'lodash/isUndefined';
+import { assert } from './utils/asserts';
+import { amBlue, amRed } from './helpers/player';
+
+let tileInfo;
+let tileNames;
+
+/*
+ * Stores all information we need about tiles in the tileInfo object, and creates the tileNames
+ * object so that we can quickly map from an id to a name. This function depends on amBlue and
+ * amRed, so the current player must be defined. We call this function once, after our player has an
+ * id.
+ */
+export function computeTileInfo() {
+  tileInfo = {
+    EMPTY_SPACE: { id: 0, traversable: false },
+    SQUARE_WALL: { id: 1, traversable: false },
+    ANGLE_WALL_1: { id: 1.1, traversable: false },
+    ANGLE_WALL_2: { id: 1.2, traversable: false },
+    ANGLE_WALL_3: { id: 1.3, traversable: false },
+    ANGLE_WALL_4: { id: 1.4, traversable: false },
+    REGULAR_FLOOR: { id: 2, traversable: true },
+    RED_FLAG: { id: 3, traversable: amRed() },
```

Comment:
All flags should now be considered fully traversable by both teams.

This change will affect `RED_FLAG` and `BLUE_FLAG`.